disperser/dataapi/server.go

```diff
@@ -402,6 +419,54 @@ func (s *server) FetchOperatorsNonsigningPercentageHandler(c *gin.Context) {
 	c.JSON(http.StatusOK, metric)
 }
 
+// FetchDeregisteredOperators godoc
+//
+//	@Summary	Fetch list of operators that have been deregistered for days
```

Comment:
Can you add to the summary that the # of days is variable and that the default is 14 and max is 30